api/ChattO/API/DTOs/Requests/User/UpdateUserRequest.cs

```diff
@@ -1,16 +1,29 @@
 ﻿using API.DTOs.Sorting;
+using Application.AppUsers.Commands;
 using Application.Helpers.Mappings;
+using AutoMapper;
 using Domain.Models;
 
 namespace API.DTOs.Requests.User;
 
-public class UpdateUserRequest 
+public class UpdateUserRequest : IMapWith<UpdateAppUser.Command>
 {
     public string? DisplayName { get; set; }
 
     public string? Email { get; set; }
 
     public string? UserName { get; set; }
 
-    public IFormFile? ProfileImage { get; set; } 
+    //public IFormFile? ProfileImage { get; set; } 
```

Comment:
why commented

Reply:
We can't updatethe picture through the PATCH method.